### ISSUE TITLE
Store images in the ghcr.io container registry

### ISSUE THREAD

**Issue by zaporylie** [nymedia/chrome-headless#1]
With Docker sunsetting Free Team accounts we're moving over to ghcr.io for the time being.

**Comment by eiriksm** [comment]
Hm, or maybe we should look into using an actual new and supported container, and sunset this image altogether

**Comment by zaporylie** [comment]
Maybe? I was under impression that it will take time to solve that so was looking for the lowest hanging fruit (aka buying ourselves time).

**Comment by eiriksm** [comment]
Fair point. I guess we need a version available at least, when the docker hub one gets deleted
